client/deposits/instant-deposits/modal.js

```diff
@@ -0,0 +1,83 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import { Button, Modal } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { __experimentalCreateInterpolateElement as createInterpolateElement } from 'wordpress-element';
+import Currency from '@woocommerce/currency';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import InstantDepositSubmitNotice from './notice';
+
+const currency = new Currency();
```

Comment:
No need to fix this now, but let's add a TODO or an issue to use the proper WCPay currency. I assume `new Currency()` defaults to `USD` when initialized without any params?

Reply:
Added the TODO.
It does look like it will do USD if nothing is passed: https://github.com/woocommerce/woocommerce-admin/blob/main/packages/currency/src/index.js